platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/SolrClientAdapter.java

```diff
@@ -786,15 +794,22 @@ boolean wasNotRecent(AtomicLong previous, long freq) {
   }
 
   private SolrPingResponse backgroundPing() throws SolrServerException, IOException {
-    return ping("in the background");
+    return ping(false, null);
   }
 
   @SuppressWarnings("squid:S1181" /* bubbling out VirtualMachineError */)
-  private SolrPingResponse ping(String how) throws SolrServerException, IOException {
+  private SolrPingResponse ping(boolean fromApi, String collection)
+      throws SolrServerException, IOException {
+    String how = fromApi ? "from the API" : "in the background";
```

Comment:
how is this get used? beside just being logged?

Reply:
Just for logging.  I was trying to keep the logic similar to how it was before.  Though now that you ask, I could probably pull that logging up into the calling methods.